src/index.html

```diff
@@ -17,6 +17,84 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="MOYO"
```

Comment:
The `alt` attribute should describe the image content. Instead of 'MOYO', describe the image content in more detail. For example, 'MOYO logo'.